dockstore-webservice/src/main/resources/swagger.yaml

```diff
@@ -6935,6 +6935,11 @@ definitions:
         format: "int64"
         position: 5
         description: "Number of workflows inside this collection"
+      notebooksLength:
```

Comment:
Since swagger doesn't get updated anymore, I think we should remove this addition to the swagger.yaml. If I understand correctly, this change occurred before you merged the develop branch to this branch